playbooks/commontemplatesbundle.yaml

```diff
@@ -1,5 +1,5 @@
 - hosts: localhost
   gather_facts: no
-  tasks:
-  - import_role:
-      name: "KubevirtCommonTemplatesBundle"
+  roles:
+  - KubevirtRepoInfo
```

Comment:
The upstream version should document this, but we must keep the role disabled to make sure the defaults for NFD containers are used

Reply:
OK. What'ts the best way to disable the role?

Reply:
Hmm I would say comment it out with a proper explanation :)

Reply:
Ack